fh-messaging/lib/mongo_functions/mbaas_reduce_apprequestsdest.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable*/
+//ts and emit are used for testing purposes
+var ts;
+var emit;
+function mbaas_reduce_apprequestsdest(k, mbaasMetrics) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this 'mbaas_reduce_apprequestsdest' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)